internal/errors/file_test.go

```diff
@@ -0,0 +1,159 @@
+package errors
+
+import (
+	"testing"
+)
+
+func TestErrWatchDirNotFound(t *testing.T) {
+	type want struct {
+		want error
+	}
+	type test struct {
+		name       string
+		want       want
+		checkFunc  func(want, error) error
+		beforeFunc func()
+		afterFunc  func()
+	}
+	defaultCheckFunc := func(w want, got error) error {
+		if !Is(got, w.want) {
+			return Errorf("got: \"%#v\",\n\t\t\t\twant: \"%#v\"", got, w.want)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "returns error",
+			want: want{
+				want: New("fs watcher watch dir not found"),
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(tt *testing.T) {
+			if test.beforeFunc != nil {
+				test.beforeFunc()
+			}
+			if test.afterFunc != nil {
+				defer test.afterFunc()
+			}
+			if test.checkFunc == nil {
+				test.checkFunc = defaultCheckFunc
+			}
+
+			got := ErrWatchDirNotFound
+			if err := test.checkFunc(test.want, got); err != nil {
+				tt.Errorf("error = %v", err)
+			}
+		})
+	}
+}
+
+func TestErrFileAlreadyExists(t *testing.T) {
+	type args struct {
+		path string
+	}
+	type want struct {
+		want error
+	}
+	type test struct {
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, error) error
+		beforeFunc func(args)
+		afterFunc  func(args)
+	}
+	defaultCheckFunc := func(w want, got error) error {
+		if !Is(got, w.want) {
+			return Errorf("got: \"%#v\",\n\t\t\t\twant: \"%#v\"", got, w.want)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "returns error when the file path is 'metadata.json'",
```

Comment:
```suggestion
			name: "returns ErrFileAlreadyExists error with the file path is 'metadata.json'",
```
I think it would be better. How do you think ? @kevindiu @hlts2

Reply:
@vankichi 
I agree with your opinion. It looks good to me.
Because even if the test fails in the ci or local environment, the test case name is clear, so I think we can respond immediately.

Reply:
I think both ok for me. Both test case name is clear enough, and the number of test case is small.

Reply:
Thank you for your comment. And I agree with you.
Just in case, I tried to output the test results of two patterns.
I want to decide which is better from the output.

***return ErrFileAlreadyExists error with the file path is 'metadata.json'***
```go
--- FAIL: TestErrFileAlreadyExists (0.00s)
    --- FAIL: TestErrFileAlreadyExists/return_ErrFileAlreadyExists_error_when_the_file_path_is_'metadata.json' (0.00s)
        file_test.go:109: error = got: "&errors.errorString{s:"file already exists: metadata.jsona"}",
            				want: "&errors.errorString{s:"file already exists: metadata.json"}"
```

***return error with the file path is 'metadata.json'***
```go
--- FAIL: TestErrFileAlreadyExists (0.00s)
    --- FAIL: TestErrFileAlreadyExists/return_error_when_the_file_path_is_'metadata.json' (0.00s)
        file_test.go:109: error = got: "&errors.errorString{s:"file already exists: metadata.jsona"}",
            				want: "&errors.errorString{s:"file already exists: metadata.json"}"

```

Which do you think is better?

Reply:
I think both ok for me. there is not much difference between them for me.
But maybe @vankichi suggestion is a little bit more clear